### DISCUSSION TITLE
Any reason not to update cameras to latest firmware via app?

### DISCUSSION THREAD

**Discussion by tensiondriven** [gtxaspec/wz_mini_hacks#158] [Q&A]
I have a small fleet of Wyze cams, all running wz_mini_hacks (thank you @gtxaspec ) - They're all on the stock firmware currently, and I'm tired of the nags I get in the Wyze app.

What I'm wondering is - Given that wz_mini_hacks seems to work with all firmwares, is there any reason /not/ to update cameras to the latest firmware via the app?

**Comment by virmaior** [comment]
versions after approximately 4.36.2.5 crash if not connected to the internet and more specifically phone-home to wyze.

so if you're not concerned about running them w/o connectivity to wyze, I'm not aware of any other issues.

**Reply by tensiondriven** [reply]
Thanks, that's very helpful!  Similarly, are there any advantages to upgrading from stock?  Sorry if the question is naive.  I imagine I could check my versions against latest and then check release notes to get a comprehensive understanding of what new features might have been added - just curious if there are any major features worth noting.

Also, if i remember correctly, it is always possible to revert to an earlier firmware, is that correct?

Hopefully having these answers here will serve as a benefit to others in the future.

**Reply by endertable** [reply]
Alot of times, they update because of new accessories, e.g. garage opener, spotlights, lamp sockets, etc. and because of security vulnerabilities, e.g. any FW before Mar 2022 can run a webserver that allows anyone to access anything on your SD card as there is no security for that webserver.

So if you don't have any accessories connected to your V3 cam and trust your LAN to be secure and are off the grid, then there really isn't a critical reason to run the latest updates if the older ones work they way you want them to.  
On the other hand, if you use the Wyze App to view your cams or have CamPlus, or you have full time internet connection to/from your cams, then the latest FW will ensure the best security of your feeds and cams.
